src/screens/PartsCCScreen.js

```diff
@@ -0,0 +1,80 @@
+/* Parts screen - re-usable screen for the Parts List for just the Criminal Code Legislation
+ */
+
+import React, {useState, useEffect} from 'react';
+import {View, Text, useWindowDimensions} from 'react-native';
+import Reactotron from 'reactotron-react-native';
+import {FlashList} from '@shopify/flash-list';
+
+//USER Imports
+//import {CCDATAPARTS, CCDATASECTION, CCSampleTest} from '../data/dummy-data'; //for DEVELOPMENT Purposes
+import styles, {colors} from '../assets/styles';
+import {CrimCodePartsList} from '../components/CrimCodeGridList';
+import {db} from '../components/Database';
+
+const PartsCCScreen = props => {
+  const [isLoading, setIsLoading] = useState(false); //for loading spinner
+  const [distinctPartsList, setDistinctPartsList] = useState();
+  const window = useWindowDimensions();
+
+  useEffect(() => {
+    getDbData();
+  }, []);
+
+  const getDbData = () => {
+    db.transaction(tx => {
+      tx.executeSql(
+        "SELECT DISTINCT heading1label, heading1titletext, heading1key FROM CCDataV2 WHERE heading1key <> '115011' and heading1key <> '115006' ORDER BY heading1key",
+        [],
+        (_tx, results) => {
+          let temp = [];
+          for (let i = 0; i < results.rows.length; ++i) {
+            temp.push(results.rows.item(i));
+          }
+          setDistinctPartsList(temp);
+        },
+      );
+    });
+  };
+
+  const renderList = itemdata => {
+    return (
+      <View>
+        <CrimCodePartsList
```

Comment:
Regarding my comment on genericizing the CrimCodeGridList - Like if this was still:
```          
<CrimCodeGridList
    heading1key={itemdata.item.heading1key}
    heading2key={itemdata.item.heading2key}
    heading1label={itemdata.item.heading1label}
    heading1titletext={itemdata.item.heading1titletext}
    firstSectionLabel={itemdata.item.firstSectionLabel}
    heading2titletext={itemdata.item.heading2titletext}
  />;
```
I think even if a prop is missing, it'll still render so for your parts, you can still send it as:
```
<CrimCodeGridList
          heading1key={itemdata.item.heading1key}
          heading1label={itemdata.item.heading1label}
          heading1titletext={itemdata.item.heading1titletext}
        />
```

Reply:
You are right.  I will work on generesizing it right now.  This issue I have to figure out is the navigation.  I'll have to do if statements to check to see which was the previous screen so the navigation to the next screen is correct.

Reply:
Done